package.json

```diff
@@ -56,8 +56,9 @@
   },
   "dependencies": {
     "@babel/runtime": "7.16.3",
-    "core-js": "^3.2.1",
-    "@kitware/vtk.js": "^24.0.0"
+    "@kitware/vtk.js": "^24.0.0",
+    "config-point": "^0.4.5",
```

Comment:
config-point as a dependency is required in order to load the sub-modes correctly, but isn't used anywhere unless you actually have dynamic loading enabled.